cmake/modules/FindSmctemp.cmake

```diff
@@ -0,0 +1,33 @@
+#.rst:
+# FindSmctemp
+# -------
+# Finds the smctemp library
+#
+# This will define the following imported targets::
+#
+#   SMCTEMP::SMCTEMP   - The smctemp library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_SMCTEMP libsmctemp QUIET)
+endif()
+
+find_path(SMCTEMP_INCLUDE_DIR NAMES smctemp.h
+                           PATHS ${PC_SMCTEMP_INCLUDEDIR} NO_CACHE)
+find_library(SMCTEMP_LIBRARY NAMES smctemp
+                          PATHS ${PC_SMCTEMP_LIBDIR} NO_CACHE)
+
+set(SMCTEMP_VERSION ${PC_SMCTEMP_VERSION})
```

Comment:
As there isnt a pkg-config file, and the pkg_check_module wont ever return anything, this will always be empty. Just remove it.